app/api/gql/datastore/datastore_user.go

```diff
@@ -0,0 +1,52 @@
+package datastore
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	oneEntity "github.com/teamyapp/one/entity"
+	"github.com/teamyapp/teamy-backend/app/entity"
+)
+
+//
+// User
+//
+func (d DataStore) GetUser(id oneEntity.ID) (entity.User, error) {
+	user, ok := d.data.Users[id]
+	if !ok {
+		return entity.User{}, fmt.Errorf("user %v is not found", id)
+	}
+	return user, nil
+}
+
+func (d DataStore) GetUsers(ids []oneEntity.ID) (users []entity.User, err error) {
+	for _, id := range ids {
+		user, ok := d.data.Users[id]
+		if ok {
+			if user.ID == id {
+				users = append(users, user)
+			} else {
+				err = errors.Errorf("user key %v and id %v doesn't match", id, user.ID)
+				return
+			}
```

Comment:
```suggestion
			}
```

To keep it consistent with tasks.